packages/at_chops/lib/src/algorithm/at_algorithm.dart

```diff
@@ -23,6 +23,20 @@ abstract class SymmetricEncryptionAlgorithm extends AtEncryptionAlgorithm {
   Uint8List decrypt(Uint8List encryptedData, {InitialisationVector iv});
 }
 
+/// Interface for asymmetric encryption algorithms. Check [DefaultEncryptionAlgo] for sample implementation.
+abstract class ASymmetricEncryptionAlgorithm extends AtEncryptionAlgorithm {
+  AtPublicKey? atPublicKey;
+  AtPrivateKey? atPrivateKey;
+
+  /// Encrypt [plainData] with [atPublicKey.publicKey] if set. Otherwise use default encryption public key set in at_chops instance
```

Comment:
This abstract class doesn't know anything about an at_chops instance

Reply:
changed the dart doc